### PR TITLE
add web UI project

### PR DESCRIPTION
**Pull request by serhiiyasenev**

### CODE REVIEW

File: src/BusinessLayer/Models/Inbound/ProductInbound.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class ProductInbound
         public float Price { get; set; }
 
         [DataType(DataType.ImageUrl)]
-        [Url(ErrorMessage = "Image URL must be a valid URL")]
+        //[Url(ErrorMessage = "Image URL must be a valid URL")]
```
Comment (YNauholnykh) [inline src/BusinessLayer/Models/Inbound/ProductInbound.cs:24]: Do you need to keeep commented out code?

File: tests/IntegrationTests/Controllers/ProductsUIControllerViewIntegrationTests.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        private readonly string _requestUri;
+        private readonly HttpClient _httpClient;
+        private readonly WebApplicationFactory<WebUI.Startup> _factory;
+
+        public ProductsUIControllerViewIntegrationTests()
+        {
+            _requestUri = "/Products";
+            _factory = new WebApplicationFactory<WebUI.Startup>()
+                .WithWebHostBuilder(builder => 
+                builder.ConfigureServices(services =>
```
Comment (YNauholnykh) [inline tests/IntegrationTests/Controllers/ProductsUIControllerViewIntegrationTests.cs:28]: Please fix indentation by shifting this statement on one tab (line 28-34)

File: tests/IntegrationTests/Controllers/ProductsUIControllerViewIntegrationTests.cs [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            var response = await _httpClient.GetAsync($"{_requestUri}/Create");
+            response.EnsureSuccessStatusCode();
+            var stringResponse = await response.Content.ReadAsStringAsync();
+            var requestVerificationToken = GetRequestVerificationToken(stringResponse);
+
+            // Arrange
+            var productName = $"Book {DateTime.UtcNow.Ticks}";
+            var keyValues = new List<KeyValuePair<string, string>>
+            {
+            new KeyValuePair<string, string>("Name", productName),
```
Comment (YNauholnykh) [inline tests/IntegrationTests/Controllers/ProductsUIControllerViewIntegrationTests.cs:60]: Please fix indentation here as well (line 60-65)

File: src/BusinessLayer/Models/Inbound/ProductInbound.cs (hunk 1 above)
Comment (serhiiyasenev) [reply src/BusinessLayer/Models/Inbound/ProductInbound.cs:24]: done

File: tests/IntegrationTests/Controllers/ProductsUIControllerViewIntegrationTests.cs (hunk 3 above)
Comment (serhiiyasenev) [reply tests/IntegrationTests/Controllers/ProductsUIControllerViewIntegrationTests.cs:60]: done

File: tests/IntegrationTests/Controllers/ProductsUIControllerViewIntegrationTests.cs (hunk 2 above)
Comment (serhiiyasenev) [reply tests/IntegrationTests/Controllers/ProductsUIControllerViewIntegrationTests.cs:28]: done

**Pull request merged**